src/Template/GoldenEye.WebApi.SimpleDDD/Backend/Issues/Issue.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Contracts.Issues;
+using GoldenEye.Backend.Core.DDD.Aggregates;
+using GoldenEye.Backend.Core.DDD.Events;
+using GoldenEye.Shared.Core.Objects.General;
+
+namespace Backend.Issues
+{
+    public class Issue : IAggregate
+    {
+        object IHasId.Id => Id;
+
+        public Guid Id { get; private set; }
+
+        public IssueType Type { get; private set; }
+
+        public string Title { get; private set; }
+
+        public string Description { get; private set; }
+
+        public Guid StreamId => Id;
```

Comment:
This should be removed, it's leftover from the `IEvent` interface implementation